### PR TITLE
Rr/review modal jp

### PR DESCRIPTION
**Pull request by Jplier**

### CODE REVIEW

File: client/components/GlobalComponents/DropDownMenu.jsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const DropDownMenu = (props) => {
     )
   }
 
+
```
Comment (kimhonrada) [inline client/components/GlobalComponents/DropDownMenu.jsx:22]: @Jplier is this a typo?

**Comment by Jplier** [comment]
https://trello.com/c/JYkAiB2U/9-user-should-be-able-to-write-and-submit-new-reviews

**Review by ZloNAGit — APPROVED** [review]
Great work as always!

**Review by kimhonrada — APPROVED** [review]
Looks good! Ready for merge

**Pull request merged**